changelog/december2023/2023-12-19-messaging-and-queuing-added-sqs-supports-json-transport.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: SQS supports JSON transport protocol 1.0
+status: added
+author:
+    fullname: 'Join the #messaging-queuing channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-12-19
+category: serverless
+product: messaging-and-queuing
+---
+
+Messaging and Queuing SQS now supports JSON transport protocol 1.0,  which is the default transport of recent SQS SDK. You can now update the SDK to the latest version.
+
+The query protocol is still supported, and you can use SDK's previous versions or custom implementation using the query transport protocol.
```

Comment:
```suggestion
Messaging and Queuing now includes support for JSON transport protocol 1.0, which serves as the default transport for the latest SQS SDK. Feel free to upgrade your SDK to the most recent version.

The query protocol remains a supported option and you have the flexibility to use the SDK's earlier versions or a custom implementation using the query transport protocol.
```

Reply:
Thanks @bene2k1 ! Great updates on my poor copy.
Though for technical reasons I'd rather keep the Messaging and Queuing SQS at the beginning.
Ok for you ?